backend/de.metas.async/src/main/java/de/metas/async/api/IWorkPackageQueue.java

```diff
@@ -219,4 +171,16 @@
 	 * @throws UnsupportedOperationException if this queue was not created with {@link IWorkPackageQueueFactory#getQueueForEnqueuing(Properties, Class)}.
 	 */
 	String getEnquingPackageProcessorInternalName();
+
+	IWorkPackageBuilder newWorkPackage();
+
+	IWorkPackageBuilder newWorkPackage(Properties ctx);
+
+	IQuery<I_C_Queue_WorkPackage> createQuery(Properties workPackageCtx, Integer limit);
+
+	void setupWorkPackageContext(Properties workPackageCtx,I_C_Queue_WorkPackage workPackage);
+
+	List<QueuePackageProcessorId> getQueuePackageProcessorIds();
```

Comment:
why not having a Set instead of List. Is the order important?

Reply:
well, nope, the order is not important, the reason we have a list here is simply bc that's how it was before this commit... changed it now to `Set`